packages/typespec-test/test/authoring/generated/typespec-ts/review/authoring.api.md

```diff
@@ -846,7 +862,10 @@ export interface SupportedLanguageOutput {
 }
 
 // @public
-export type SupportedLanguagesOutput = Paged<SupportedLanguageOutput>;
+export interface SupportedLanguagesOutput {
```

Comment:
@joheredi @qiaozha There is a change in this pr which I believe it is not a breaking and I think it's better to have, could you help review this?

The changes are:
- Paged interface is removed
- The page body model is changed from `Paged<SupportedLanguageOutput>` to directly model defined in TypeSpec.